lib/README.md

```diff
@@ -24,3 +24,8 @@ That's it!
 2. Run in watch mode `npm run test:watch` 
 3. Run end-to-end test `npm run test:e2e` 
 | Keep in mind that these tests are slow as they run against true servers. And therefore, it cannot be guaranteed that all E2E tests always work
```

Comment:
And e2e would need some additional setup, which I'll describe in a later readme update